target/linux/pistachio/base-files/lib/netifd/wireless/mac80211.sh

```diff
@@ -0,0 +1,759 @@
+#!/bin/sh
```

Comment:
We should avoid having such big drops in the target. I think we have this file somewhere already. Please propose a patch where the original file was taken from.